FDBLibTLS/local.mk

```diff
@@ -1,6 +1,14 @@
+
+ifeq ($(PLATFORM),freebsd)
+FDBLibTLS_CFLAGS := -fPIC -I/usr/local/include -I$(BOOSTDIR)
+FDBLibTLS_LIBS := -lssl -lcrypto
+FDBLibTLS_LDFLAGS := -lc++
```

Comment:
I believe link-wrapper re-writes -static-libstdc++ to something else in the command line anyway to manage glibc version dependency issues.

Putting no thought yet into libc version dependency issues, could you drop all your local.mk edits by adding an `if freebsd` there, or doing something similar that looks more platform-agnostic?  The next time we add a folder, we're likely to accidentally break FreeBSD by forgetting to have a separate `if freebsd` block for it, or not being able to test it even if we remember.

Reply:
It was dropped when I sorted out that last conflict to be added back in again for the next check-in to see if we can get this to build correctly, I'm comparing the working version (FDBLibTLS built against FreeBSD) I have in that other repo with this.